plugins/browser-plugin-media-tracking/src/findMediaElement.ts

```diff
@@ -0,0 +1,38 @@
+export function findMediaElem(mediaId: string): HTMLAudioElement | HTMLVideoElement | null {
+  let el: HTMLVideoElement | HTMLAudioElement | HTMLElement | null = document.getElementById(mediaId);
+
+  if (el instanceof HTMLVideoElement) {
+    // Plyr loads in an initial blank video with currentSrc as https://cdn.plyr.io/static/blank.mp4
+    // so we need to check until currentSrc updates
+    if (el.src === 'https://cdn.plyr.io/static/blank.mp4' || !el.src) {
```

Comment:
This doesn't work if you load a video like:

```
  <video id="playerid" playsinline controls>
    <source src="test-video.mp4" type="video/mp4" />
  </video>
```

I think we need to be looking at `currentSrc`.
```suggestion
    if (el.currentSrc === 'https://cdn.plyr.io/static/blank.mp4' || !el.currentSrc) {
```

My below sample should also switch to currentSrc too